homeassistant/components/onkyo/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: exempt
+    comment: |
+      This integration uses a push API. No polling required.
+  brands: done
+  common-modules: done
+  config-flow:
+    status: todo
+    comment: |
+      The data_descriptions are missing.
+  config-flow-test-coverage:
+    status: todo
+    comment: |
+      Coverage is 100%, but the tests need to be improved.
+  dependency-transparency: done
+  docs-actions: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration do not explicitly subscribe to events.
```

Comment:
To talk about the flow of data, I think we should improve the integration not to use `hass.data` anymore since we should not keep a reference of the created entities after they are created.